app/controllers/application_controller.rb

```diff
@@ -26,6 +27,10 @@ def reviewer?
     @is_reviewer ||= current_user.reviewer?
   end
 
+  def event_staff?
+    event.event_teammates.where(user_id: current_user.id).length > 0
+  end
+
```

Comment:
You might play around with CanCanCan to see if you can just move this into the ability.rb file.